src/components/CustomSelect.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+
+export default function CustomSelect({ label, value, options, onChange }) {
+  return (
+    <label>
+      {label}
+      <select value={value} onChange={onChange}>
+        {options.map((option) => (
+          <option value={option.value}>{option.label}</option>
+        ))}
+      </select>
+    </label>
+  );
+}
+
```

Comment:
In prop types you can mark property as required, this is very important. Now all your props are optional, is it correct? If yes, you can add CustomSelect.defaultProps for all optional props